authLdap.php

```diff
@@ -12,7 +12,7 @@
 require_once ABSPATH . 'wp-includes/registration.php';
 
 function authLdap_debug($message) {
-    if (get_option('authLDAPDebug')) {
+    if (authLdap_getOptions()['Debug']) {
```

Comment:
The current min requirement for Wordpress is currently 5.2.4 (Yes, I know it's absolutely outdated and the recommended Version is 5.4 but....) But the functionality you are using here is only available from 5.4 on. So it would be a BC-break.

Can you update the footprint of ```authLdap_getOptions()``` to optionaly take a parameters that specifies which option to get? That would then read ```authLdap_getOptions('Debug')``` and return the corresponding value.